code_generation/data/attribute_classes/input.json

```diff
@@ -127,6 +127,33 @@
                 }
             ]
         },
+        {
+            "name": "GenericBranchInput",
+            "base": "BranchInput",
+            "attributes": [
+                {    
+                    "data_type": "double",
+                    "names": ["r1","x1","g1","b1"],
+                    "description": "positive sequence parameters"
+                },
+                {
+                    "data_type": "double",
+                    "names": "k",
+                    "description": "transformer ratio, default = 1.0"
+                },
+                {
+                    "data_type": "double",
+                    "names": "theta",
+                    "description": "angle shift"
```

Comment:
We do not use degree anywhere in PGM interface. Please emphasize in the description that this is in radian.